runtime/common/src/weights/pallet_parameters.rs

```diff
@@ -46,15 +46,11 @@ use sp_std::marker::PhantomData;
 /// Weights for `pallet_parameters`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_parameters::WeightInfo for WeightInfo<T> {
-	/// Storage: `Parameters::Parameters` (r:1 w:1)
-	/// Proof: `Parameters::Parameters` (`max_values`: None, `max_size`: Some(36), added: 2511, mode: `MaxEncodedLen`)
 	fn set_parameter() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `3`
-		//  Estimated: `3501`
-		// Minimum execution time: 5_480_000 picoseconds.
-		Weight::from_parts(5_610_000, 3501)
-			.saturating_add(T::DbWeight::get().reads(1_u64))
-			.saturating_add(T::DbWeight::get().writes(1_u64))
```

Comment:
If I remember correctly, the benchmark does not account PoV in this call because we whitelist the storage since it is already accounted for in the `on_initialize` hook of pallet parameters